tox.ini

```diff
@@ -25,3 +29,4 @@ commands = flake8
 max-line-length = 160
 ignore = E302,E251
 max-complexity = 10
+exclude = venv,.git,__pycache__,.tox
```

Comment:
Again, a bunch of directories to ensure flake8 doesn't dig into stuff we don't care about.